SampleMultiplayerClient/MultiplayerClient.cs

```diff
@@ -129,6 +136,21 @@ Task IMultiplayerClient.UserStateChanged(int userId, MultiplayerUserState state)
             return Task.CompletedTask;
         }
 
+        public Task MatchUserStateChanged(int userId, MatchUserState state)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task MatchRoomStateChanged(MatchRoomState state)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task MatchEvent(MatchServerEvent e)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Maybe implement? You've at least implemented changing the type in `Program.cs`.

Reply:
I kind of gave up on this because I found it pretty non-useful for testing due to database requirements (plan to redo this at a later stage). I'll add the bare minimum though untested.